pkg/build/strategies/sti/sti.go

```diff
@@ -198,6 +221,47 @@ func (builder *STI) Prepare(config *api.Config) error {
 		WorkingDir: config.WorkingDir,
 	}
 
+	if len(config.RuntimeImage) > 0 {
+		if config.Incremental {
+			return fmt.Errorf("Incremental build with runtime image isn't supported")
```

Comment:
this should have been sorted out earlier when validating the CLI options in main.go.